docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Steps from '../_partials/_get-started-steps.md';
+
+This article is an introduction to using stream workers with the Macrometa GDN console.
+
+<Prerequisites />
+
+## Get Started with Stream Workers
+
+This page guides you through creating a stream worker and updating it.
+
+<Steps />
+
+If you want to skip the explanation and just run the code, then go directly to the [Full Demo File](#full-demo-file).
+
+### Step 1. Define Query Name
+
+Write a name and description. If no `qlVersion` is specified, default is `2`. Do not change this value unless you are working with a version 1 stream.
```

Comment:
I don't think `version 1 stream` gives the correct idea. Could you change it to `stream worker written in query language version 1` or something to give that idea?
cc: @StoyanTc

Reply:
I think this should be changed like this: `The default qlVersion is 2 if not specified, qlVersion='1' is only for backward compatibility and is deprecated.`